pandas/tests/window/test_rolling.py

```diff
@@ -1082,28 +1080,133 @@ def test_rolling_sem(frame_or_series):
 
 
-@pytest.mark.xfail(
-    is_platform_arm() or is_platform_power() or is_platform_riscv64(),
-    reason="GH 38921",
-)
 @pytest.mark.parametrize(
-    ("func", "third_value", "values"),
+    ("func", "values", "window", "ddof", "exp_value"),
     [
-        ("var", 1, [5e33, 0, 0.5, 0.5, 2, 0]),
-        ("std", 1, [7.071068e16, 0, 0.7071068, 0.7071068, 1.414214, 0]),
-        ("var", 2, [5e33, 0.5, 0, 0.5, 2, 0]),
-        ("std", 2, [7.071068e16, 0.7071068, 0, 0.7071068, 1.414214, 0]),
+        (
+            "var",
+            [
+                2.72993945,
+                1.58444294,
+                4.14371708,
+                4.92961687,
+                2.7138744,
+                3.48168586,
+                0.69505519,
+                1.87511994,
+                4.20167276,
+                0.04797675,
+            ],
+            3,
+            1,
+            "numpy_compute",
+        ),
+        (
+            "std",
+            [
+                2.72993945,
+                1.58444294,
+                4.14371708,
+                4.92961687,
+                2.7138744,
+                3.48168586,
+                0.69505519,
+                1.87511994,
+                4.20167276,
+                0.04797675,
+            ],
+            3,
+            1,
+            "numpy_compute",
+        ),
+        (
+            "var",
+            [
+                2.72993945,
+                1.58444294,
+                4.14371708,
+                4.92961687,
+                2.7138744,
+                3.48168586,
+                0.69505519,
+                1.87511994,
+                4.20167276,
+                0.04797675,
+            ],
+            2,
+            1,
+            "numpy_compute",
+        ),
+        (
+            "std",
+            [
+                2.72993945,
+                1.58444294,
+                4.14371708,
+                4.92961687,
+                2.7138744,
+                3.48168586,
+                0.69505519,
+                1.87511994,
+                4.20167276,
+                0.04797675,
+            ],
+            2,
+            1,
+            "numpy_compute",
+        ),
+        ("var", [99999999999999999, 1, 1, 2, 3, 1, 1], 2, 1, 0),
+        ("std", [99999999999999999, 1, 1, 2, 3, 1, 1], 2, 1, 0),
+        ("var", [99999999999999999, 1, 2, 2, 3, 1, 1], 2, 1, 0),
+        ("std", [99999999999999999, 1, 2, 2, 3, 1, 1], 2, 1, 0),
+        ("var", [1, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 0], 5, 0, "numpy_compute"),
     ],
 )
-def test_rolling_var_numerical_issues(func, third_value, values):
-    # GH: 37051
-    ds = Series([99999999999999999, 1, third_value, 2, 3, 1, 1])
-    result = getattr(ds.rolling(2), func)()
-    expected = Series([np.nan] + values)
-    tm.assert_series_equal(result, expected)
+def test_rolling_var_correctness(func, values, window, ddof, exp_value):
+    # This tests subsume the previous tests under test_rolling_var_numerical_issues
```

Comment:
Super nit but you can remove this first comment - if the user cared to see the history they can consult the git blame